utils/build-script-impl

```diff
@@ -2012,7 +2012,6 @@ for host in "${ALL_HOSTS[@]}"; do
                     -DCLANG_TOOL_ARCMT_TEST_BUILD=NO
                     -DCLANG_TOOL_C_ARCMT_TEST_BUILD=NO
                     -DCLANG_TOOL_C_INDEX_TEST_BUILD=NO
-                    -DCLANG_TOOL_DRIVER_BUILD=NO
```

Comment:
build-toolchain-only and build-runtime-using-host-compiler=0 are not compatible without this patch. (We need the clang driver for that)- since other configs are broken without this too, I'd vote for this as well. I tried building for release without cross compiling and it worked FWIW.